### PR TITLE
add crcmod

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

File: recipes/crcmod/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    name: '{{ name|lower }}'
+    version: '{{ version }}'
+
+source:
+    fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+    '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+    number: 0
```
Comment (CJ-Wright) [inline recipes/crcmod/meta.yaml]: Could this be a `noarch` package?

File: recipes/crcmod/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    script: 'python setup.py install'
```
Comment (CJ-Wright) [inline recipes/crcmod/meta.yaml]: Is there an issue with `python setup.py install --single-version-externally-managed --record record.txt`

Comment (bgruening) [reply recipes/crcmod/meta.yaml]: I thought this is just needed if setuptools is used.

File: recipes/crcmod/meta.yaml (hunk 1 above)
Comment (bgruening) [reply recipes/crcmod/meta.yaml]: I added noarch, thanks!

File: recipes/crcmod/meta.yaml (hunk 2 above)
Comment (CJ-Wright) [reply recipes/crcmod/meta.yaml]: Ah that is most likely true, although it may also be required with noarch?

File: recipes/crcmod/meta.yaml (hunk 1 above)
Comment (bgruening) [reply recipes/crcmod/meta.yaml]: Does not seem to work.

Comment (CJ-Wright) [reply recipes/crcmod/meta.yaml]: Oh well, thank you for trying!

**Pull request merged**